src/analytics/Properties.tsx

```diff
@@ -834,6 +835,22 @@ interface FiatExchangeEventsProperties {
     isLowestFee: boolean | undefined
   } & ProviderSelectionAnalyticsData
   [FiatExchangeEvents.cico_providers_back]: { flow: CICOFlow }
+  [FiatExchangeEvents.cico_providers_fetch_quotes_result]: {
+    fiatType: LocalCurrencyCode
+    defaultFiatType: LocalCurrencyCode
+  } & Omit<ProviderSelectionAnalyticsData, 'transferCryptoAmount'> &
+    (
+      | {
+          flow: CICOFlow.CashOut
+          cryptoAmount: number // given by user
+          fiatAmount: undefined // exchange rate varies by provider
```

Comment:
Any reason to include an explicit undefined?

Reply:
yeah, otherwise I found that if I left the amount in it still passed the type check. it's misleading to consumers of the data to have a non null fiatAmount for cash outs (and cryptoAmount for cash ins) because in reality, that amount varies by provider. so any query that uses it would be buggy